From 595fc1e7caa0ab1fe582885851bd54ce75e17220 Mon Sep 17 00:00:00 2001
From: Alvin Wong <alvin@alvinhc.com>
Date: Wed, 14 Dec 2022 19:31:53 +0800
Subject: [PATCH] Use vendored copy of winmd

---
 CMakeLists.txt | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b5f547fd..7ef85896 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -177,8 +177,7 @@ endif()
 
 include(ExternalProject)
 ExternalProject_Add(winmd
-    GIT_REPOSITORY https://github.com/microsoft/winmd.git
-    GIT_TAG 0f1eae3bfa63fa2ba3c2912cbfe72a01db94cc5a
+    SOURCE_DIR "${PROJECT_SOURCE_DIR}/winmd"
     CONFIGURE_COMMAND ""
     BUILD_COMMAND ""
     INSTALL_COMMAND ""
-- 
2.24.1.windows.2

